app/helpers/application_helper.rb

```diff
@@ -55,7 +55,7 @@ def datetime_to_js_ms(utc_string)
 
   def sortable(column, title = nil)
     title ||= column.titleize
-    direction = (column == sort_column && sort_direction == "asc") ? "desc" : "asc"
+    direction = (column == params[:sort] && params[:direction] == "asc") ? "desc" : "asc"
```

Comment:
This helper methods were specific to the User model, and were defined in the UsersController. So, here, we ended up with what was supposed to be a cross-application helper tight to this specific use case of sorting the Users:
```
def sort_column
    User.column_names.include?(params[:sort]) ? params[:sort] : "created_at"
  end

  def sort_direction
    %w[asc desc].include?(params[:direction]) ? params[:direction] : "asc"
  end
```
I needed this `sortable` helper method for the projects as well, so I needed to make this `sortable` method agnostic of any model. There's a tradeoff here, though: I'm assuming whoever uses this will supply the correct column names. Before, by using the helper methods, we were falling back to a default column provided by the helper method. However, for the existing scenarios, it was not a problem as we're checking the sort columns in the controller before executing the query against the database.